go.mod

```diff
@@ -0,0 +1,13 @@
+module go_Playlist_gRPC
```

Comment:
It's a bad practice to use `_` and upper letters in the module name. Please check how it's done for a usual Go project placed on GitHub.